index.ts

```diff
@@ -1,3 +1,21 @@
+// Global error handler
+process.on('uncaughtException', (err) => {
+  console.error('Uncaught Exception:', err);
+  process.exit(1); // Optional: exit the process after logging the error
```

Comment:
Is the point of this to gracefully shut down the server on error? Not sure how it works - I'm supposing the idea is that you:
1. log the error
2. shutdown the server gracefully 
3. rely on the docker instructions to restart the server

Is that the case?